content/english/android-tictactoe/_index.md

```diff
@@ -1,249 +1,22 @@
 ---
 title: "Android: Tictactoe"
 description: "Learn about variables, functions, and conditionals all while building a Tic-Tac-Toe Android app."
-date: 2019-07-23T14:28:42-07:00
-prereq: "link to workshop markdown file"
-difficulty: "Beginner, Intermediate, or Advanced"
-download: "URL to GitHub folder goes here"
+date: 2020-07-27T14:04:59.61-07:00
+prereq: "Android Studio"
+difficulty: "Intermediate"
+download: "Github URL determined prior to publish"
```

Comment:
I think this should be the link to download the files.

Reply:
Agreed, it says the link should point to the Github location. Should I add an URL with the assumption that the files are sitting in the "master" branch?

Reply:
Temporarily adding a relative link to it. Can change to direct Github link later on.